testing/endtoend/components/eth1/transactions.go

```
@@ -136,30 +161,80 @@ func SendTransaction(client *rpc.Client, key *ecdsa.PrivateKey, f *filler.Filler
 	if expectedPrice.Cmp(gasPrice) > 0 {
 		gasPrice = expectedPrice
 	}
+
+	// Check if we're near or post-Fulu fork
+	currentSlot := slots.CurrentSlot(e2e.TestParams.CLGenesisTime)
+	currentEpoch := slots.ToEpoch(currentSlot)
+	fuluForkEpoch := params.BeaconConfig().FuluForkEpoch
+
+	// Stop sending blob transactions 1 epoch before Fulu fork to avoid execution client bug at fork boundary
+	nearFuluFork := fuluForkEpoch > 0 && currentEpoch == (fuluForkEpoch-1)
+	isPostFulu := currentEpoch >= fuluForkEpoch
+
 	g, _ := errgroup.WithContext(context.Background())
 	txs := make([]*types.Transaction, 10)
-	for i := uint64(0); i < 10; i++ {
-		index := i
-		g.Go(func() error {
-			tx, err := RandomBlobTx(client, f, fundedAccount.Address, nonce+index, gasPrice, chainid, al)
-			if err != nil {
-				logrus.WithError(err).Error("Could not create blob tx")
-				// In the event the transaction constructed is not valid, we continue with the routine
-				// rather than complete stop it.
-				//nolint:nilerr
+
+	// Send blob transactions - skip near fork boundary, use different versions pre/post Fulu
+	if nearFuluFork && !isPostFulu {
+		logrus.Infof("Near Fulu fork boundary (epoch %d, fork at %d): Skipping blob transactions to avoid execution client bug", currentEpoch, fuluForkEpoch)
+	} else if isPostFulu {
+		logrus.Info("Post-Fulu: Sending blob transactions with Version 1 sidecars (cell proofs)")
+		// Reduced from 10 to 5 to conserve funds during extended test runs
+		for i := uint64(0); i < 5; i++ {
+			index := i
+
+			g.Go(func() error {
+				tx, err := RandomBlobCellTx(client, f, fundedAccount.Address, nonce+index, gasPrice, chainid, al)
+				if err != nil {
+					logrus.WithError(err).Error("Could not create blob cell tx")
```

What happens if we cannot correctly send a tx with blobs?
Does the e2e test will fail?

My fear is we always silently ignore the error, the e2e test is green, and at the end if Prysm has an issue related with blobs post fulu nobody will notice.

---

hmm good question I guess i can return it or panic or soemthing, before it was the evaluators should pick it up, i guess in this case we are testing geth?

---

I left the old blob ones the way they are